packages/modules/src/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -103,15 +105,23 @@ export const ContributionsEpicButtons = ({
     submitComponentEvent,
     isReminderActive,
     isSignedIn,
+    choiceCardSelection,
 }: ContributionsEpicButtonsProps): JSX.Element | null => {
     const [hasBeenSeen, setNode] = useHasBeenSeen({}, true);
-
     const { cta, secondaryCta, showReminderFields } = variant;
 
     if (!cta) {
         return null;
     }
 
+    const getCta = (cta: Cta): Cta =>
```

Comment:
this is fine for now, but later we should refactor how urls are built so that all these things are done in one place, and shared by primary/secondary ctas

Reply:
Yeah we need to add region + tracking to this url don't we?

Reply:
Ahhh I see, this is being added first, and then e.g `PrimaryCtaButton` will change it again

Reply:
Yeah we should refactor that

Reply:
yes, and that is still done inside `PrimaryCtaButton` and `SecondaryCtaButton`. It might be better for the full urls to be constructed here, but I think it can be considered beyond the scope of this PR